src/locales/fr/translation.json

```
@@ -35,6 +35,9 @@
     "titleButtonBackAlt": "Retour à la liste",
     "defaultCameraName": "Camera inconnue",
     "buttonImageDownload": "Télécharger",
+    "buttonPlay": "Play",
+    "buttonPause": "Pause",
+    "errorFetchImagesMessage": "Une erreur est survenue sur le serveur. Veuillez réessayer plus tard.",
```

prop :
```suggestion
    "errorFetchImagesMessage": "Une erreur est survenue. Veuillez réessayer plus tard.",
```

et perso j'aime bien les messages du genre "notre équipe est sur le coup", mais je sais pas si on a du monitoring sur le back donc on verra plus tard ^^

---

We have the same message on every error of this type (http 500). But I could change it in every place. The user doesn't need to know where the problem comes from (server or something else). I will change it